pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -282,8 +283,14 @@ stdenv.mkDerivation {
       if [ -e $ldPath/${prefix}as ]; then
         ln -s $ldPath/${prefix}as $out/bin/${prefix}as
       fi
-
-      wrap ${prefix}ld ${preWrap ./ld-wrapper.sh} ''${ld:-$ldPath/${prefix}ld}
+    '' + (if !useMacosReexportHack then ''
+      wrap ${prefix}ld ${./ld-wrapper.sh}
+    '' else ''
+      wrap ${prefix}ld ${./ld-wrapper.sh}
+      export ldWrapper=${prefix}ld
+      wrap ${prefix}ld-reexport-delegate ${./macos-sierra-reexport-hack.sh}
```

Comment:
Looks like `reexport-delegate` isn't referenced anywhere but from this script itself, how does this get called? :confused: